debian/rules

```diff
@@ -276,46 +64,57 @@ override_dh_auto_install:
 	cp /usr/share/fonts/truetype/ubuntu/Ubuntu-R.ttf \
 		debian/tmp/usr/share/fonts/Plymouth.ttf
 
-	dh_auto_install --destdir=debian/tmp --buildsystem=meson+ninja --sourcedirectory=vendor/systemd
-	for p in $(CLEAN_PATHS); do		\
-		rm -r debian/tmp/$$p ;		\
-	done
 	touch debian/tmp/etc/machine-id
+
+	# Include all needed systemd bits
+	set -e;				\
+	for f in $$(dpkg -L systemd |	\
+	grep -E '(^/lib/systemd/system|modprobe\.d/|sysctl\.d/|rules\.d/|tmpfiles\.d/|bin/)')	\
```

Comment:
There is no `tmp.mount` unit as far as I can see. Agreed that we should opt-in services instead of opt-out. We can do that as a follow up. I've added the `/` where needed.

Reply:
Ah! There is actually a bug in the ubuntu package. It is installed in `/usr/share/systemd/tmp.mount` instead of `/usr/share/systemd/system/tmp.mount`. It has been broken for a long time. So we do not need to work-around this one for now.